crates/services/gas_price_service/src/ports.rs

```diff
@@ -12,3 +16,13 @@ pub trait L2Data: Send + Sync {
         height: &BlockHeight,
     ) -> StorageResult<Option<Block<Transaction>>>;
 }
+
+pub trait MetadataStorage: Send + Sync {
```

Comment:
Why do we want to move this trait here? I thought that each service(v0 and v1) would have its own ports. It is the same for storage traits as well.

Reply:
The metadata storage would remain the same since we discussed that we will keep it versioned right

Reply:
It was up to you=D I was thinking that you will choose the approach with two different columns, since it is more flexible.